src/status.h

```diff
@@ -57,13 +57,18 @@ class Status {
     NetSendErr,
   };
 
-  Status() : Status(cOK, "ok") {}
-  explicit Status(Code code, std::string msg = "") : code_(code), msg_(std::move(msg)) {}
+  Status() : Status(cOK) {}
```

Comment:
@ShooterIT @git-hulk may I ask why `cOK` gets its name at the first place instead of `OK`?

Reply:
because we already have a method named `OK` 🤣

Reply:
> because we already have a method named `OK` 🤣

Yes, exactly. But `cOK` is NOT a good name since it's inconsistent with other code.